app/bundles/EmailBundle/Controller/PublicController.php

```diff
@@ -63,8 +63,10 @@ public function indexAction(Request $request, AnalyticsHelper $analyticsHelper,
                 $content = $copy->getBody();
 
                 // Replace tokens
-                $content = str_ireplace(array_keys($tokens), $tokens, $content);
-                $subject = str_ireplace(array_keys($tokens), $tokens, $subject);
+                if (!empty($tokens)) {
```

Comment:
Ok, if it is a common practice then let's get this merge. One suggestion, can we make it 
```suggestion
                if (is_array($tokens)) {
```
like it is on line 56 for consistency and really making sure that it won't fail if it is different type for whatever reason?

Reply:
Done with the changes. Thank you for the suggestion.